tests/bigdata-test.c

```diff
@@ -84,7 +84,7 @@ static uint32_t hash_big_text()
     hash = fnv1a(hash, "it was taken off the market, research showed that brand");
     hash = fnv1a(hash, "recognition for the model was still at 87%.");
 
-    return hash;
+    return hash = 2659567138;
```

Comment:
what is it testing?

Reply:
The same as before, there is just typo, it should be `return hash == 2659567138;`.